step-functions/nino_issue_credential.asl.json

```diff
@@ -346,7 +391,10 @@
         "Entries": [
           {
             "Detail": {
-              "message": "VC Successfully Issued"
+              "msTimestamp.$": "$.currentTime.milliseconds",
+              "sTimestamp.$": "$.currentTime.seconds",
+              "session.$": "$.session",
+              "issuer.$": "$.parameters.issuer"
```

Comment:
In other VC issued events I've seen we've included `ci` and `ciReasons` if there are any. I can't see any payloads on the tickets but I wondered if we wanted to do that here? Maybe it's a question for Sanka?

Reply:
The main goal of this ticket was to setup the infrastructure, and at the moment the payload for the events just cover all of the base properties required for TXMA. Follow up tickets will add further properties when we have had discussions with TXMA on exactly what they want for each event.